runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -400,6 +406,8 @@ NNFW_STATUS nnfw_session::load_model_from_nnpackage(const char *package_dir)
       model->bindKernelBuilder(_kernel_registry->getBuilder());
       _nnpkg->push(onert::ir::ModelIndex{i}, std::move(model));
       _coptions.push_back(onert::compiler::CompilerOptions::fromGlobalConfig());
+      // TODO load TrainingInfo from model, using TraininfoLoader
+      _train_info = onert::ir::train::TrainingInfo::createDefaultInfo();
```

Comment:
I have a plan about TODO comment like this. 
A draft code related with this TODO - https://github.com/Samsung/ONE/pull/12152/files#r1426184845